Formula/php-cs-fixer.rb

```diff
@@ -1,9 +1,9 @@
 class PhpCsFixer < Formula
   desc "Tool to automatically fix PHP coding standards issues"
   homepage "https://cs.sensiolabs.org/"
-  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.0/php-cs-fixer.phar"
-  version "2.16.0"
-  sha256 "59a5f7fe8c6e946fdd7426c39e11d502c48549b1c79e87ddebdbf9d4c022c194"
+  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.1/php-cs-fixer.phar"
+  version "2.16.1"
```

Comment:
isn't the `version` superfluous?

works fine without:
- https://github.com/Homebrew/homebrew-core/pull/46550/files

Reply:
Looks like Homebrew's version detection function detects `cs` as the version from this particular URL:

```ruby
> Version.detect("https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.16.1/php-cs-fixer.phar", {})
=> #<Version::FromURL:0x00007fc08209cc58 @version="cs">
```

Reply:
Homebrew can guess the version from the __file name__, but not from the path. The file name does not contain the version, thus we need to specify it.